### DISCUSSION TITLE
docker pull error

### DISCUSSION THREAD

**Discussion by LawrenceUy** [jdetras/SNP-Calling#11] [Q&A]
![image](https://user-images.githubusercontent.com/73359114/202394405-a7e28e5c-bc6e-4dd8-82c1-03ba2150efa9.png)

Hello! 

I can't pull the image locally.

**Comment by jdetras** [comment]
Hi,

Can you try this: nextflow run jdetras/snp-calling -r main -profile docker. (without 1.0)

**Comment by LawrenceUy** [comment]
Noted kuya. Thank you po.
